### DISCUSSION TITLE
R5.2.0 Sunbird Knowledge BB Signoff

### DISCUSSION THREAD

**Discussion by Naveenkumar739** [Sunbird-Knowlg/Community#83] [General Discussions]
Hello All,

Please find the Sunbird knowledge Building block R5.2.0 sign-off here.

Environment: https://dev.knowlg.sunbird.org/

Testing Status: Completed

Defects Dashboard: [link](https://project-sunbird.atlassian.net/jira/software/c/projects/KN/issues/?jql=project%20%3D%20%22KN%22%20AND%20reporter%20IN%20%28currentUser%28%29%2C%2260f117ba43c956007055794e%22%29%20AND%20type%20IN%20%28%22Defect%22%2C%22Checklist-Item%22%29%20ORDER%20BY%20created%20DESC)

Execution Sheet: [link
](https://docs.google.com/spreadsheets/d/16Zb9lZw9gLGdzugKF8gHlkoDW-vDMD0amY7afJ5wbT0/edit#gid=690730285)

Scope : [Link](https://project-sunbird.atlassian.net/issues/?filter=11287&jql=project%20%3D%20KN%20AND%20issuetype%20in%20standardIssueTypes()%20AND%20status%20in%20(Done%2C%20%22Failed%20Validation%22%2C%20%22In%20Development%22%2C%20%22In%20Validation%22%2C%20%22To%20Do%22)%20AND%20Sprint%20in%20(273%2C%20274)%20ORDER%20BY%20key%20ASC%2C%20created%20DESC)

BB Sign Off Template: [Link](https://docs.google.com/spreadsheets/d/1mu_liuEDb_x1JBABS6PYxQW5aPY5STNMJnb76eD7ejw/edit#gid=0)

cc @vinukumar-vs @anunagaraj @ShailandraOjha @chitranshu-keshav

**Comment by anunagaraj** [comment]
@vinukumar-vs Please confirm if the migration scripts handover is done to the Devops team as part of the sign off.
@gandham-santhosh [keshavprasadms](https://github.com/keshavprasadms) @G33tha Please confirm if the migration scripts are received.

CC @rhwarrier @Krishnaj20 @sharathkashyap @chitranshu-keshav @ashokreddy1208 @ShailandraOjha

**Reply by vinukumar-vs** [reply]
@AmiableAnil How to communicate the job & configuration changes for migration. We can discuss this in today's call with [@keshavprasadms](https://github.com/keshavprasadms).

Especially configuration changes & script in the job.

**Reply by anunagaraj** [reply]
@vinukumar-vs Please share an update on the above comment as we will not be able to start the regression until we have the confirmation on the migration scripts handed over to DevOps to run in the Sunbird ed environment.

CC @rhwarrier @Krishnaj20 @sharathkashyap @chitranshu-keshav @ashokreddy1208 @ShailandraOjha [@Santhosh Gandham](mailto:santhosh.g@optit.in) [@Keshav](mailto:keshav.prasad@tarento.com) @G33tha

**Comment by anunagaraj** [comment]
@vinukumar-vs The scope signed off for R5.2 is not there in the release notes of Knwlg microsite. Please update the microsite with the scope signed off.

CC @ShailandraOjha @Krishnaj20  @sharathkashyap @chitranshu-keshav @rhwarrier

**Reply by anunagaraj** [reply]
@vinukumar-vs @Naveenkumar739 There are only 7 ticket id's (KN-316,KN-523,KN-517,KN-304,KN-603,KN-546,KN-547)  mentioned as part of R5.2 [Release notes](https://knowlg.sunbird.org/use/release-notes/release-5.2.0-ongoing) but as per Jira and the sign off provided there are 35 tickets. Please confirm on the tickets which have been . Please note the tickets for which sign off is provided should match with the tickets mentioned in the Microsite and the tickets signed off .

5.2 Scope for Knowlg as per Jira - [https://project-sunbird.atlassian.net/issues/?filter=12697](https://project-sunbird.atlassian.net/issues/?filter=12697)

CC CC @ShailandraOjha @Krishnaj20 @sharathkashyap @chitranshu-keshav @rhwarrier

**Reply by vinukumar-vs** [reply]
@anunagaraj Release notes should be useful for any adopter to know what are the primary/key features that particular release is having. We will not document each and every jira item in the release notes. For these you can go through the JIRA link shared in Release notes/Discussion thread(QA sing-off). 

@Naveenkumar739 @Krishnaj20  @chitranshu-keshav @ShailandraOjha

**Reply by anunagaraj** [reply]
@Krishnaj20 Being a Sunbird Knwlg Adopter .How will you get to know the features available for this release to be adopted unless you are having the complete list of the Release. Also how will the QA team be notified what are the features adopted by Sunbird ED.

@chitranshu-keshav @ShailandraOjha

**Reply by Krishnaj20** [reply]
@vinukumar-vs @anunagaraj - The release notes should capture the key features or bugs getting addressed in that release. But should also provide link to a detailed list of other fixes that are in-scope for that release. This is how druid does - pls check this link: https://github.com/apache/druid/releases
